packages/gestalt/src/DropdownItem.jsdom.test.js

```diff
@@ -0,0 +1,60 @@
+// @flow strict
+import React from 'react';
+import { create } from 'react-test-renderer';
+
+import { render } from '@testing-library/react';
+import Dropdown from './Dropdown.js';
+import Icon from './Icon.js';
+
+describe('DropdownItem', () => {
+  const handleSelectMock = jest.fn();
+  test('calls handleSelect when Item clicked', () => {
+    const { getByText } = render(
```

Comment:
The current best practice is to not use the queries from `render`, but from `screen`: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen